instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/OpenTelemetryDriver.java

```diff
@@ -99,8 +102,48 @@ public static boolean isRegistered() {
     return REGISTERED.get();
   }
 
-  private static Driver findDriver(String realUrl) {
-    for (Driver candidate : Collections.list(DriverManager.getDrivers())) {
+  /**
+   * Add driver candidate that wasn't registered against {@link DriverManager}. This is mainly
+   * useful when drivers are initialized at native image build time, and you don't want to postpone
+   * class initialization (f.e. runtime initialization could make driver incompatible with other
```

Comment:
i'm not sure what `f.e.` stands for?
```suggestion
   * class initialization (f.e. runtime initialization could make driver incompatible with other
```